packages/extension/src/view/devtools/components/cookies/cookiesListing/index.tsx

```diff
@@ -36,7 +36,6 @@ import {
  * Internal dependencies.
  */
 import { useCookieStore } from '../../../stateProviders/syncCookieStore';
-import { BLOCKED_REASON_LIST } from '../../../../../constants';
```

Comment:
I see no other references to this constant in the project. Maybe we can delete it.